### ISSUE TITLE
[Bug] - Error immediately after update to version 0.12.4

### ISSUE THREAD

**Issue by Gamma518** [Monika-After-Story/MonikaModDev#8114]
-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:/WINDOWS/system32/00updater.rpyc", line 23, in script
  File "C:\Users\XXX\OneDrive\desktop\Game\DDLC\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\XXX\OneDrive\desktop\Game\DDLC\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/00updater.rpy", line 121, in <module>
    process_deferred()
  File "renpy/common/00updater.rpy", line 104, in process_deferred
    l = l.decode("utf-8")
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/encodings/utf_8.py", line 16, in decode
UnicodeDecodeError: 'utf8' codec can't decode byte 0x94 in position 36: invalid start byte

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.4

**Comment by ThePotatoGuy** [comment]
start with [this FAQ entry](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue). If you still have issues, create a new issue.
